kroxylicious-schema-validation/src/main/java/io/kroxylicious/proxy/filter/schema/JsonValueValidator.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.filter.schema;
+
+import java.nio.ByteBuffer;
+
+import org.apache.kafka.common.record.Record;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.kroxylicious.proxy.filter.schema.validation.RecordValidator;
+
+/**
+ * Checks if a Record's value is valid JSON by attempting to deserialize it
+ * with jackson
```

Comment:
Jackson is an implementation detail, so I don't think you really need to mention it here, but if you do then get the capitalization right.

```suggestion
 * with Jackson
```